app/assets/stylesheets/_forms.scss

```diff
@@ -88,6 +88,21 @@ label {
   }
 }
 
+.form-field {
+  label {
+    line-height: 40px;
```

Comment:
As I said in https://github.com/instedd/cdx/pull/1543#issuecomment-1069044354, fixing `line-height` to an absurdly large value is not a proper solution. It breaks as soon as the line breaks:
![grafik](https://user-images.githubusercontent.com/466378/159468235-3b48a150-f7b7-4ea8-96d1-05601fc95d51.png)

`line-height` controls the distance between the baselines of consecutive lines of texts. It's usually just a multiplier. In this case, it should be something around 1.6.
We actually want to control the distance from the top of the cell to the top of the text. That's `padding` or `margin`.

In combination with the adjusted `line-height`, a top margin of 9px should work well:
```suggestion
    line-height: 1.6;
    margin-top: 9px;
```
![grafik](https://user-images.githubusercontent.com/466378/159469193-d71ab59a-c83a-47cd-a8d4-cc8407934c56.png)